### PR TITLE
Script updating a modern slavery statement

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/vTkIJ4WK/2103-ccsrequests-uploading-a-new-modern-slavery-statement

This script implements the process: https://alphagov.github.io/digitalmarketplace-manual/2nd-line-runbook/support-tasks.html?#adding-or-updating-a-modern-slavery-statement

This should make it faster and less error-prone in future.

Once this is merged, I intend to use it for a real user request to prove that it works. I'll then update the manual.

### CODE REVIEW

**Review by lfdebrux — APPROVED** [review]
🙌

File: scripts/upload-modern-slavery-statement.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    file_path = arguments['<file_path>']
+    supplier_id = arguments['<supplier_id>']
+    framework = arguments['<framework>']
+    s3_file_name = arguments['<s3_file_name>']
+    stage = arguments['<stage>'] or 'production'
+
+    bucket_name = f"digitalmarketplace-documents-{stage}-{stage}"
+    s3_key_name = f"{framework}/documents/{supplier_id}/{s3_file_name}"
+    asset_url = f"https://www.digitalmarketplace.service.gov.uk/suppliers/{s3_key_name}"
```
Comment (alex9smith) [inline scripts/upload-modern-slavery-statement.py:45]: Should this be?
```suggestion
    asset_url = f"https://assets.digitalmarketplace.service.gov.uk/suppliers/{s3_key_name}"
```

Comment (gidsg) [inline scripts/upload-modern-slavery-statement.py:45]: Are we sure this is correct?
The format in the example in the manual is:
 ```
"https://assets.digitalmarketplace.service.gov.uk/<framework_slug>/documents/<int:supplier_id>/<filename>"
```

File: scripts/upload-modern-slavery-statement.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+from dmutils.s3 import S3
+import sys
+
+sys.path.insert(0, '.')
+from dmscripts.helpers.auth_helpers import get_auth_token
+
+
+def get_slavery_statement_declaration_key(declaration):
+    if declaration['declaration']['modernSlavery']:
+        return 'modernSlavery'
```
Comment (lfdebrux) [inline scripts/upload-modern-slavery-statement.py:29]:
```suggestion
        return 'modernSlaveryStatement'
```

**Review by lfdebrux — CHANGES_REQUESTED** [review]

File: scripts/upload-modern-slavery-statement.py (hunk 1 above)
Comment (gidsg) [reply scripts/upload-modern-slavery-statement.py:45]: @alex9smith I don't _think_ it should include `/suppliers/`.
Please see example in the manual.

Comment (alex9smith) [reply scripts/upload-modern-slavery-statement.py:45]: I think you're correct

Comment (bjgill) [reply scripts/upload-modern-slavery-statement.py]: Oops. Thanks for noticing. Fixed.

Comment (bjgill) [reply scripts/upload-modern-slavery-statement.py]: Oops. Thanks for noticing. Fixed.

File: scripts/upload-modern-slavery-statement.py (hunk 2 above)
Comment (bjgill) [reply scripts/upload-modern-slavery-statement.py]: Oops. Thanks for noticing. Fixed.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**